tools/README.md

```diff
@@ -149,6 +149,13 @@ node ./bin/3d-tiles-tools.js optimizeB3dm -i ./specs/data/batchedWithBatchTableB
 |`-f`, `--force`|Overwrite output file if it exists.| No, default `false` |
 |`--options`|All arguments past this flag are consumed by gltf-pipeline.| No |
 
+To use tileset texture compression, pass the [`texcomp` flags](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/README.md#command-line-flags)
+from [gltf-pipeline](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/README.md) under the `--options` flag.
+```
+node ./bin/3d-tiles-tools.js optimizeB3dm -i ./specs/data/Textures/batchedTextured.b3dm -o ./output/optimized.b3dm --options --texcomp.dxt1.enable --texcomp.dxt1.quality=5 --texcomp.etc1.enable
+```
+This example optimizes the b3dm and compresses the textures into `dxt1` and `etc1` formats.
+
```

Comment:
Place this example after the other examples above. This part of the description is not needed: `from [gltf-pipeline](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/README.md) under the --options flag.` since it should be clear from the first example.